server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/exchanges/coinzix/dto/entities/Order.java

```diff
@@ -0,0 +1,17 @@
+package com.generalbytes.batm.server.extensions.extra.bitcoin.exchanges.coinzix.dto.entities;
+
+public class Order {
+    public long id;
+    public int type;
+    public int status;
+    public int type_trade;
+    public String pair;
+    public double volume;
```

Comment:
Do you really want to use `double` for volume/price/rate? Have you considered using precise BigDecimal instead?

Reply:
Thank you for response. We do not use those properties while processing order. We are using property 'status' to determine order execution status. We changed Order properties from double to String to avoid situation with wrong serialization of big numbers and null for future.

Reply:
OK. In that case I would also recommend to change it at `OrderBookPrice`

Reply:
Thank you. Changed type of properties `rate` and `volume` of `OrderBookPrice` to String and all appearances in code are updated.